test/src/901-printchangeset/main

```diff
@@ -0,0 +1,128 @@
+  ln /cvmfs/$CVMFS_TEST_REPO/new_file2.txt /cvmfs/$CVMFS_TEST_REPO/new_file4.txt || return 7
+  ls -li /cvmfs/$CVMFS_TEST_REPO/new_file4.txt /cvmfs/$CVMFS_TEST_REPO/new_file2.txt || return 7
+  let ADD_COUNTER++
+  touch /cvmfs/$CVMFS_TEST_REPO/new_file3.txt || return 8
+  ln -s /cvmfs/$CVMFS_TEST_REPO/new_file3.txt /cvmfs/$CVMFS_TEST_REPO/new_file5.txt || return 8
+  ls -l /cvmfs/$CVMFS_TEST_REPO/new_file5.txt || return 8
+  let ADD_COUNTER++
+  let ADD_COUNTER++
+  echo "*** Publish transaction for testing addition of entries"
+  sudo cvmfs_server publish -v $CVMFS_TEST_REPO | tee logfile1.txt || return 9
+
+  add_value=$(grep -w "add" logfile1.txt | wc -l)
+  if [ $ADD_COUNTER != $add_value ]
+  then
+    echo "There is a missmatch between the number of additions printed and the ones performed" && return 9
+  fi
+
+  echo "*** Start transaction for testing modification of entries"
+  start_transaction $CVMFS_TEST_REPO || return 10
+  touch /cvmfs/$CVMFS_TEST_REPO/new_folder2 || return 11
+  let MOD_COUNTER++
+  echo "*** Publish transaction for testing modification of entries"
+  sudo cvmfs_server publish -v $CVMFS_TEST_REPO | tee logfile2.txt || return 12
+
+  mod_value=$(grep -w "mod" logfile2.txt | wc -l)
+  if [ $MOD_COUNTER != $mod_value ]
+  then
+    echo "There is a missmatch between the number of modifications printed and the ones performed" && return 12
+  fi
+
+  echo "*** Start transaction for testing removal of entries"
+  start_transaction $CVMFS_TEST_REPO || return 20
+  rm /cvmfs/$CVMFS_TEST_REPO/new_folder/.cvmfscatalog || return 21
+  let REM_COUNTER++
+  let REM_COUNTER++
+  rm /cvmfs/$CVMFS_TEST_REPO/new_file.txt || return 23
+  let REM_COUNTER++
+  rmdir /cvmfs/$CVMFS_TEST_REPO/new_folder2 || return 24
+  let REM_COUNTER++
+  echo "*** Publish transaction for testing removal of entries"
+  sudo cvmfs_server publish -v $CVMFS_TEST_REPO | tee logfile3.txt || return 25
+
+  rem_values=$(grep -w "rem" logfile3.txt | wc -l)
+  if [ $REM_COUNTER != $rem_values ]
+  then
+    echo "There is a missmatch between the number of deletions printed and the ones performed" && return 25
+  fi
+
+  echo "*** Start transaction for testing printing of dots in additons"
+  start_transaction $CVMFS_TEST_REPO || return 30
+  echo "*** Create a hundred of folders"
+  for ii in {0..200}
+  do
+    mkdir /cvmfs/$CVMFS_TEST_REPO/"folder${ii}" || return 31
+  done
+  echo "*** Publish transaction for testing printing of dots in additons"
+  sudo cvmfs_server publish $CVMFS_TEST_REPO | tee logfile4.txt || return 32
+
+  add_dots=$(grep -x ".." logfile4.txt | wc -l)
+  echo "Number of dots detected $add_dots"
+  if [ $add_dots != 1 ]
+  then
+    echo "There is a missmatch between the number of files added and the changes printed" && return 32
+  fi
+  
+
+  echo "*** Start transaction for testing printing of dots in modifications"
+  start_transaction $CVMFS_TEST_REPO || return 40
+  echo "*** Modify a hundred of folders"
+  for ii in {0..200}
```

Comment:
Actually, we might need more.  Let's use 500. Because we use `...` in the output in any case.